.github/workflows/docker_publish.yml

```diff
@@ -96,16 +26,22 @@ jobs:
           clang,
           ]
         hdf5_versions : [
-          1.10.4,
+         1.10.4,
+        ]
+        moab_versions : [
+         5.3.0,
+         develop,
+         master,
+        ]
+        static_exe : [
+         OFF,
+         ON,
```

Comment:
We don't appear to be passing this into the docker build or the tag naming and therefore have duplicate images being built and tagged.  We should discuss whether we really need the statically linked version.  Not sure I remember who requires/desires that.

Reply:
static_exe has been removed from the matrix so I think this is resolved